cmd/cmd.go

```diff
@@ -117,6 +115,10 @@ var globalFlags = []cli.Flag{
 		Value:  "",
 		EnvVar: "BUCKET",
 	},
+	cli.BoolFlag{
+		Name:  "no-multipart",
```

Comment:
please, rename to `s3-no-multipart`
also remember to add documentation in `README.md`

Reply:
Done